app/presenters/order_detail_presenter.rb

```diff
@@ -0,0 +1,64 @@
+class OrderDetailPresenter < SimpleDelegator
+
+  include ActionView::Helpers::NumberHelper
+  include DateHelper
+
+  delegate :admin_editable?, to: :reservation, prefix: true
+  delegate :name, to: :order_status, prefix: true
+  delegate :template_result, to: :stored_files, prefix: true
+
+  def self.wrap(order_details)
+    order_details.map { |order_detail| new(order_detail) }
+  end
+
+  def actual_total
+    format_as_currency(__getobj__.actual_total)
+  end
+
+  def description_as_html
+    [bundle, product].compact.map do |description|
+      ERB::Util.html_escape(description)
+    end.join(" &mdash; ").html_safe
+  end
+
+  def edit_reservation_path
+    edit_facility_order_order_detail_reservation_path(facility, order, id, reservation)
```

Comment:
Good question, and the answer was a surprise to me. The `OrderDetail` model includes `NotificationSubject`, which then includes `url_helpers`, which feels like something a decorator/presenter could make unnecessary.

I'll add the `include` explicitly here in the presenter, since I don't want to count on that sticking around.